content/webapp/pages/article.tsx

```diff
@@ -270,7 +272,8 @@ const ArticlePage: FC<Props> = ({ article }) => {
           const nextArticle = articles.find(i => i.title === nextArticleTitle);
 
           const nextUp =
-            positionInSerial === series.schedule.length
+            positionInSerial === series.schedule.length &&
+            series.schedule.length > 1
```

Comment:
Is it worth pulling this into a method that we can test, so we don't regress this later?

e.g. `chooseNextUp(series, articles)`?